### DISCUSSION TITLE
Prevent multiple instances of the app running

### DISCUSSION THREAD

**Discussion by tmblog** [pyinstaller/pyinstaller#8218] [Help]
Hi I've created a frozen python package with pyinstaller, when I double click the exe it loads, however I can open multiple instances of the same app by clicking on the exe. 
Is there a built in way to stop the app from opening again if it already is open? e.g only allow one instance of it to run.

Thanks

**Comment by rokm** [comment]
> Is there a built in way to stop the app from opening again if it already is open?

No, there is not; you would need to implement this at the start of your program code (i.e., detect an existing instance and exit immediately).
